app/assets/javascripts/discourse/app/controllers/user-private-messages.js

```diff
@@ -1,67 +1,28 @@
 import Controller, { inject as controller } from "@ember/controller";
 import { action } from "@ember/object";
 import { alias, and, equal } from "@ember/object/computed";
-import I18n from "I18n";
-import Topic from "discourse/models/topic";
-import bootbox from "bootbox";
 import discourseComputed from "discourse-common/utils/decorators";
 import { VIEW_NAME_WARNINGS } from "discourse/routes/user-private-messages-warnings";
 
 export default Controller.extend({
-  userTopicsList: controller("user-topics-list"),
```

Comment:
This and other field changes could break any themes/plugins that rely on them. Did you do a quick search through all our repos just in case?

Reply:
Yup I did a quick search and it doesn't seem like any plugin or themes are relying on this.